packages/client/tailwindTheme.ts

```diff
@@ -153,7 +153,11 @@ export default {
         '700': '#444258',
         '800': '#2D2D39',
         '900': '#1C1C21'
-      }
+      },
+      'success-light': '#2db553',
+      starter: '#F2E1F7',
+      team: '#CBECF0',
```

Comment:
+1 I can see `success-light` being used in this PR, but not starter, team or enterprise?

Reply:
I added these just for completeness, so the old [PALETTE](https://github.com/ParabolInc/parabol/blob/feat/10024/selfHostedGitLabIntegration/packages/client/styles/paletteV3.ts#L85-L89) and the tailwind colours match.